src/components/Explorer.vue

```diff
@@ -43,6 +43,14 @@
 import store from '@/store'
 
 export default {
+	async mounted() {
```

Comment:
Let's not make this async. If you also look at the dispatched event its also async, seems to do less good. Let's not make those calls async anymore. You can also remove the try catch block